src/functionalTest/java/uk/gov/hmcts/reform/waworkflowapi/controllers/SendMessageTest.java

```diff
@@ -83,24 +85,35 @@ public void transition_creates_a_task_with_default_due_date() {
             .then()
             .statusCode(HttpStatus.NO_CONTENT_204);
 
-        waitSeconds(3);
+        AtomicReference<String> taskIdResponse = new AtomicReference<>();
+        await()
+            .ignoreExceptions()
+            .pollInterval(2, TimeUnit.SECONDS)
```

Comment:
do we want to poll a bit faster say something like 500 Milliseconds ?

Reply:
down to 1 sec. Cheers!